packages/checkbox-group/test/lion-checkbox-indeterminate.test.js

```diff
@@ -301,18 +331,20 @@ describe('<lion-checkbox-indeterminate>', () => {
     const elParentIndeterminate = /**  @type {LionCheckboxIndeterminate} */ (el.querySelector(
       '#parent-checkbox-indeterminate',
     ));
+    const ElNestedSubCheckboxes = getProtectedMembers(elNestedIndeterminate);
```

Comment:
Instance or 'regular variable' names should not be capitalized. So usually we have the convention:

```js
class MyClass {}

const myClass = new MyClass();

```

So in the example above `MyClass` is the constructor, `myClass` is the instance. As a rule of thumb, we can say that only constructor names should be capitalized, all other variable names should start with a lowercase char.